ideascube/library/static/library/library.css

```diff
@@ -0,0 +1,42 @@
+/* ************************************************* */
```

Comment:
Pure css files are now merged into one. This now add only one new request per page instead of three (for purecss).